client/components/search-result.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import SearchResultList from './search-result-list';
+
+export default class SearchResults extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```

Comment:
If you are not using the state or binding any methods, you can leave out the whole constructor.